ergo/conditions/variance.py

```diff
@@ -18,11 +18,14 @@ def __init__(self, variance, weight=1.0):
         super().__init__(weight)
 
     def actual_variance(self, dist) -> float:
-        # FIXME: Should be interacting with HistogramDist via pdf
+        # FIXME: Should be interacting with PointDensity via pdf
         #        or similar public interface
-        xs = np.linspace(dist.scale.low, dist.scale.high, dist.ps.size)
-        mean = np.dot(dist.ps, xs)
-        return np.dot(dist.ps, np.square(xs - mean))
+        """
+        xs = np.linspace(dist.scale.low, dist.scale.high, dist.normed_densities.size)
+        mean = np.dot(dist.normed_densities, xs)
+        return np.dot(dist.normed_densities, np.square(xs - mean))
+        """
```

Comment:
Remove FIXME and string. In fact, `actual_variance` doesn't even need to be a method anymore, can just call `dist.variance()` directly.